sympy/printing/pretty/pretty.py

```diff
@@ -857,6 +857,15 @@ def _print_HadamardProduct(self, expr):
         return self._print_seq(expr.args, None, None, delim,
                 parenthesize=lambda x: isinstance(x, (MatAdd, MatMul)))
 
+    def _print_KroneckerProduct(self, expr):
+        from sympy import MatAdd, MatMul
+        if self._use_unicode:
+            delim = u' \N{N-ARY CIRCLED TIMES OPERATOR} '
+        else:
+            delim = 'x'
```

Comment:
Shouldn't there be a space for the ascii printer too?

Reply:
Yes, there probably should be.